templates/common/copilot-gpt-basic/README.md

```diff
@@ -1 +1,40 @@
-For testing, will be updated soon.
\ No newline at end of file
+# Overview of the Copilot GPT template
+
+## Build a Copilot GPT from OpenAPI description document
```

Comment:
I am not sure whether we should name it as this. This template is used for two cases:
1. Basic GPT. User won't provide API description doc
2. GPT + plugin. User will provide API description for generating plugin.

Reply:
Yes, this readme is just a draft version and should be updated multiple times later. I think we should focus on the e2e testing for now. I'll have the PM review the readme, and also add some feature flags later.